helm-chart/renku/values.yaml

```diff
@@ -72,6 +72,29 @@ global:
     jena:
       ## specify which dataset should be used in Jena
       dataset:
+
+  # Connection details for a globally used redis instance for the
+  # entire platform. For specifying an actual instance as part of
+  # this chart, check out the non-global "redis" section.
+  redis:
+    # Specify which renku component should use which DB index
+    dbIndex:
+      gateway: "0"
+      # coreService: "1"
+      # uiServer: "2"
+
+    # Note: these two entries MUST match the ones in redis.auth if redis is
+    # installed through this chart.
+    existingSecret: redis-secret
+    existingSecretPasswordKey: redis-password
+
+    ## Specify the redis host. This will fall-back to
+    ## RELEASE-NAME-redis if left empty.
```

Comment:
Not sure I get how this works. I would rather define `host` here so that all the clients can just access `redis.host`

Reply:
The problem is that a value defined here will not be accessible to the sub-charts, and redis doesn't give you the possibility to define the `host` in a global variable.

Reply:
So the way it will work is that in our default use case, the redis host will not be specified through any values file. Instead, the redis chart comes with a named template called "redis.fullname". These named templates (often defined in `_helpers.tpl`) are all globally accessible during template rendering, even those which come from a sub-chart. Therefore, all the charts and subcharts can use `{{ template redis.fullname }}` and they will get the same result defined by the redis chart itself.